src/app/containers/EpisodeList/Link.jsx

```diff
@@ -51,22 +51,29 @@ const StyledAnchor = styled.a`
   }
 `;
 
-const Link = ({ children, showMediaIndicator, dir, ...props }) => {
+const Link = ({ children, showMediaIndicator, dir, index, ...props }) => {
   return (
-    <StyledAnchor showMediaIndicator={showMediaIndicator} {...props}>
+    <StyledAnchor
+      showMediaIndicator={showMediaIndicator}
+      aria-labelledby={index}
+      {...props}
+    >
       {showMediaIndicator && (
         <MediaIndicatorWrapper dir={dir}>
           <MediaIndicator size="2.5rem" />
         </MediaIndicatorWrapper>
       )}
-      <span role="text">{children}</span>
+      <span role="text" id={index}>
+        {children}
+      </span>
     </StyledAnchor>
   );
 };
 
 Link.propTypes = {
   children: node.isRequired,
   dir: string.isRequired,
+  index: string.isRequired,
```

Comment:
Is index always required, what happens when there's only one link?

Reply:
It will simply have an index of 0

<img width="1792" alt="Screenshot 2021-11-11 at 11 09 31" src="https://user-images.githubusercontent.com/90621252/141288289-4c385a84-e875-45b7-9620-d8c9bb42c494.png">
 0

Reply:
I was just wondering if `episodeLinkIndex-0` is good a11y practice as there will only be one instance of the link in that context, so does it need an index or would that confuse users

Reply:
`episodeLinkIndex-0` is not read to the user. This value is used to reference the Anchor element span child. A screen reader will then read the content of that span element.

<img width="954" alt="Screenshot 2021-11-11 at 14 16 30" src="https://user-images.githubusercontent.com/90621252/141313214-9b0565cf-4762-439c-8ac1-7ccc7c8e2dc5.png">

Reply:
That is a value for devs only

Reply:
@DarioR01 Just a quick one on this, does this need to be `index: number.isRequired,` now?

Reply:
hmmmm yes, I think that would make more sense. I will try to fix this ;-;